pkgs/development/python-modules/python-lsp-server/default.nix

```diff
@@ -75,19 +77,22 @@ buildPythonPackage rec {
   ++ lib.optional withPyflakes pyflakes
   ++ lib.optional withPylint pylint
   ++ lib.optional withRope rope
-  ++ lib.optional withYapf yapf;
+  ++ lib.optionals withYapf [ whatthepatch yapf ];
 
   checkInputs = [
     flaky
     matplotlib
     numpy
     pandas
+    pytest-cov
```

Comment:
Everything related to coverage doesn't add value for us. Please adjust `addopts = ..`.

Reply:
(To be more clear: adjust the patch that removes the coverage arguments to patch `pyproject.toml` instead of `setup.cfg`.)